qemu/tests/cfg/qemu_img_measure.cfg

```diff
@@ -0,0 +1,30 @@
+- qemu_img_measure:
+    only qcow2
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = no
+    images = "test1"
+    image_name_test1 = "images/test1"
+    variants:
+        - new_image:
+            type = qemu_img_measure_new_image
+            images += " test2"
+            image_name_test2 = "iamges/test2"
+            preallocated_test1 = off
+            preallocated_test2 = full
+        - convert_image:
+            type = qemu_img_measure_convert_image
+            write_size = 1G
+            convert_tags = "cvt1 cvt2"
```

Comment:
It would be better to name the target images as "target1 target2" or "tgt1 tgt2"